src/templates/next/components/internal/Footer/Footer.tsx

```diff
@@ -108,7 +108,7 @@ const SocialMediaSection = ({
               rel="noopener noreferrer nofollow"
               aria-label={`${link.type} page`}
             >
-              <Icon className="w-[1.625rem] h-auto" />
+              <Icon className="w-[1.5rem] h-auto" />
```

Comment:
can `w-6` instead!

Reply:
fixed in [19ad69d](https://github.com/isomerpages/isomer-components/pull/100/commits/19ad69d3762c74d88026a9132d6381c0f9ef2d48)!